Source/Applications/SystemCenter/Controllers/SystemCenter/SystemCenterController.cs

```diff
@@ -854,18 +854,18 @@ private IEnumerable<ParsedChannel> ParsePQDiff(byte[] bytes, string meterKey)
                         channelInstance.Definition.QuantityTypeID == QuantityType.ValueLog ||
                         channelInstance.Definition.QuantityTypeID == QuantityType.Phasor;
 
-                    if (!timeValueChannel)
+                    if (!timeValueChannel || channelInstance.SeriesInstances.Count() == 0)
                         continue;
 
-                    foreach (SeriesInstance seriesInstance in channelInstance.SeriesInstances.Skip(1))
-                    {
-                        // Create a channel from the parsed series instance
-                        seriesInstances.Add(seriesInstance);
-                        openXDA.Model.Channel channel = PQDIFReader.ParseSeries(seriesInstance);
-
-                        // Add the new channel to the meter's channel list
-                        parsedChannels.Add(channel);
-                    }
+                    openXDA.Model.Channel channel = PQDIFReader.ParseSeries(channelInstance.SeriesInstances[0]);
+                    channel.Series = channelInstance.SeriesInstances.SelectMany((seriesInstance) =>
+                        {
+                            seriesInstances.Add(seriesInstance);
+                            openXDA.Model.Channel tempChannel = PQDIFReader.ParseSeries(seriesInstance);
+                            return tempChannel.Series;
+                        }).ToList();
+                    // Add the new channel to the meter's channel list
+                    parsedChannels.Add(channel);
```

Comment:
This change stops the triple channels we were seeing but it changes the interpretation from each SeriesInstance having it's own channel to ChannelInstance doing so. This is a divergence from similar logic in openXDA dataReaders.

Reply:
Per Discussion with Gabriel that is resolved in `ConfigurationOperation` in the openXDA. That's not happening here which is why we had to refactor this to include that logic in some form